server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -109,6 +113,33 @@ See the radio-option below as an example.
   align-items: center;
 }
 
+.cf-prose-h2 {
+  @include u-font('sans', 'xl');
+  @include u-line-height('sans', 2);
+  font-weight: font-weight('bold');
+}
+
+.cf-prose-intro {
```

Comment:
There is already a .cf-prose-intro style defined below, we should probably put them altogether.